src/components/QuickRunParametersModal.vue

```diff
@@ -1,9 +1,7 @@
 <template>
   <p-modal v-model:showModal="internalShowModal" class="parameters-modal" title="Run Deployment">
     <p-form @submit="submit">
-      <p-content>
-        <DeploymentParameters v-model="parameters" :deployment="deployment" />
-      </p-content>
+      <SchemaInput v-model="parameters" :schema="deployment.parameterOpenApiSchema" />
```

Comment:
Should this be using the rawSchema?

Reply:
I don't think so, no - that looks like it was incorrect before since `DeploymentParameters` was stripping required properties when this form should be respecting them.

Reply:
yeah actually on another look this was happening in a couple of places but should be fixed now - the only one of these forms that needs to bypass the existing schema is the DeploymentForm, since you can leave parameters unset